### PR TITLE
Calculator branch

### PR DESCRIPTION
**Pull request by mcguireprice**
Lets make this calc

### CODE REVIEW

File: CalculatorWithTests.Test/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+        [Theory]
+        [InlineData(400, 50, 8)]
+        [InlineData(5, 1, 5)]
+        [InlineData(13, 13, 1)]
+        [InlineData(-10, 5, -2)]
+        [InlineData(-8, -2, 4)]
+        [InlineData(40, 5, 8)]
+        [InlineData(0, 5, 0)]
+        public void Divide(int a, int b, int expected)
```
Comment (chipelmer) [inline CalculatorWithTests.Test/UnitTest1.cs:76]: There's a suspiciously absent test case here, which is when the divisor is 0.  Edge cases like this should have their behavior defined by the tests

File: CalculatorWithTests.Test/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+        [Theory]
+        [InlineData(5, 5, 0)]
+        [InlineData(40, 3, 1)]
+        [InlineData(52, 5, 2)]
+        [InlineData(10, 5, 0)]
+        [InlineData(500, 3, 2)]
+        [InlineData(1, 1, 0)]
+        [InlineData(5, 1, 0)]
+        public void Factorial(int a, int b, int expected)
```
Comment (chipelmer) [inline CalculatorWithTests.Test/UnitTest1.cs:96]: This doesn't seem to be testing for factorial. Factorial is multiplying a number by every number below it. So, 5! = 5 * 4 * 3 * 2 * 1 = 120. Or, 8! = 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1 = 40,320

File: CalculatorWithTests.Test/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        }
+
+        [Theory]
+        [InlineData(1, 100, 1)]
+        [InlineData(4, 2, 16)]
+        [InlineData(10, 3, 1000)]
+        [InlineData(2, 4, 16)]
+        [InlineData(13, 5, 371293)]
+        [InlineData(3, 6, 729)]
+        public void PowerOf(int a, int b, int expected)
```
Comment (chipelmer) [inline CalculatorWithTests.Test/UnitTest1.cs:115]: Nice branching out beyond the basic requirements 👍

File: CalculatorWithTests/EmptyInterface.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿using System;
+namespace CalculatorWithTests
+{
+    public interface EmptyInterface
```
Comment (chipelmer) [inline CalculatorWithTests/EmptyInterface.cs:4]: Unused code/files should be removed

**Review by chipelmer — APPROVED** [review]
Looks great, with a few minor things such as the Factorial test.
